configs/coins/deeponion.json

```diff
@@ -0,0 +1,68 @@
+{
+  "coin": {
+    "name": "DeepOnion",
+    "shortcut": "ONION",
+    "label": "DeepOnion",
+    "alias": "deeponion"
+  },
+  "ports": {
+    "backend_rpc": 8091,
+    "backend_message_queue": 38391,
+    "blockbook_internal": 9091,
+    "blockbook_public": 9191
+  },
+  "ipc": {
+    "rpc_url_template": "http://127.0.0.1:{{.Ports.BackendRPC}}",
+    "rpc_user": "rpc",
+    "rpc_pass": "rpc",
+    "rpc_timeout": 25,
+    "message_queue_binding_template": "tcp://127.0.0.1:{{.Ports.BackendMessageQueue}}"
+  },
+  "backend": {
+    "package_name": "backend-deeponion",
+    "package_revision": "satoshilabs-1",
+    "system_user": "deeponion",
+    "version": "2.0.5",
+    "binary_url": "https://github.com/deeponion/deeponion/releases/download/v2.0.5/deeponion-2.0.5-x86_64-linux-gnu.tar.gz",
+    "extract_command": "tar -C backend --strip 1 -xf",
+    "exclude_files": [
+      "bin/DeepOnion-qt"
+    ],
```

Comment:
You need to remove `bin/DeepOnion-qt` since is not included on the file

Reply:
Should be fixed, thanks.